Source/ThirdParty/libwebrtc/Source/webrtc/sdk/objc/components/video_codec/RTCVideoDecoderH264.mm

```diff
@@ -359,6 +396,12 @@ - (void)destroyDecompressionSession {
 - (void)flush {
   if (_decompressionSession)
     VTDecompressionSessionWaitForAsynchronousFrames(_decompressionSession);
+
+  webrtc::MutexLock lock(&_reorderQueueLock);
+  while (!_reorderQueue.empty()) {
+    _callback(_reorderQueue.front()->take());
```

Comment:
is there a chance that the callback insert a new frame which would cause a deadlock?